buildworker.rst

```diff
@@ -82,40 +82,40 @@ For Mac:
     * Log in as the buildbot user.
     * Install either the Python 2.7 bundle from python.org [#]_, or pip.
     * Open a terminal window.
-    * Execute ``pip install buildbot-slave``.
+    * Execute ``pip install buildbot-worker``.
 
 For Windows:
 
     * Create a buildbot user as a "standard" user.
     * Install the latest version of Python 2.7 from python.org.
     * Open a Command Prompt.
-    * Execute ``python -m pip install pypiwin32 buildbot-slave`` (note that
+    * Execute ``python -m pip install pypiwin32 buildbot-worker`` (note that
       ``python.exe`` is not added to ``PATH`` by default, making the
       ``python`` command accessible is left as an exercise for the user).
 
 In a terminal window for the buildbot user, issue the following commands (you
 can put the ``buildarea`` wherever you want to)::
 
     mkdir buildarea
-    buildslave create-slave buildarea buildbot.python.org:9020 slavename slavepasswd
+    buildworker create-worker buildarea buildbot.python.org:9020 workername workerpasswd
```

Comment:
s/buildworker/buildbot-worker/

This applies everywhere the `buildslave` command was used.